### PR TITLE
Csv Error

### LINKED ISSUE #113: CSV file not found error
## Describe the bug
If you run the *buildserver* command and do not have the __**role_lists**__ folder an error will occur and not let you build the server
The error is in the *destroyserver* command

## To Reproduce
1. Start up the bot
2. Delete role_lists folder or double check you do not have this folder
3. Execute the buildserver command

## Expected behavior
The bot will throw a FileNotFoundException when trying to find the role_lists directory

## Screenshot
image
image

## Discord Link
[Error](https://discord.com/channels/808423775048564807/817441972134084649/971091194228637716)

## Additional context
In order to fix this problem we can make the error more pretty to the user
We could also do a test that prompts the user to skip the removing of the roles
We could also try to create a CSV from scratch

### PR DESCRIPTION
**Pull request by ReginPotter**
# Description

The code now handles the error when there is no role_lists directory instead of throwing an error. This is handled by using try-excepts

## Issues

Closes #113 

## Type of change

Select one or more of the following:

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other (describe below)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. List any edge cases you tested.
If you come up with any edge cases you didn't test while writing this PR, cancel the PR and test again.

- [ ] Test A
       - Execute command -buildserver without a role_lists directory involved

# Checklist:

- [X] All local commits have been pushed to remote
- [X] All changes on the base branch have been merged into this branch, either by rebase or merge
- [X] My code is [PEP-8](https://pep8.org/) compliant (excluding maximum line length, keep that to 100ish characters)
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have added [Google Docstrings](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) for all new functions/methods
- [X] I have made corresponding changes to the documentation

### CODE REVIEW

File: Cogs/ServerManagement.py [hunk 1]
```diff
@@ -120,8 +128,14 @@ async def destroyserver(self, ctx):
         """
 
         # Load roles csv
+
         csv_filepath = f'role_lists/roles_{ctx.guild.id}.csv'
-        roles_csvs = pd.read_csv(csv_filepath)
+
+        # If the role_lists directory doesnt exist raise a FileNotFoundError back to the buildserver command
```
Comment (mbozada) [inline Cogs/ServerManagement.py:134]: doesn't

**Review by mbozada — CHANGES_REQUESTED** [review]

**Review by mbozada — APPROVED** [review]
Nice fix '
